src/anomalib/data/video/avenue.py

```diff
@@ -212,7 +212,7 @@ def __init__(
         num_workers: int = 8,
         transform_config_train: str | A.Compose | None = None,
         transform_config_eval: str | A.Compose | None = None,
-        val_split_mode: ValSplitMode = ValSplitMode.FROM_TEST,
+        val_split_mode: ValSplitMode = ValSplitMode.SAME_AS_TEST,
```

Comment:
All image datasets except `Folder` already have this as the default behaviour. The motivation for having FROM_TEST in the Folder dataset is that `FROM_TEST` is actually a better practice, because it leads to mutually exclusive validation and test sets. The reason we use `SAME_AS_TEST` for the other datasets is for reproducibility: Results in literature are evaluated over the entire test set from which no samples are withheld for validation. Because Folder dataset always concerns custom data, we can safely create a val-test split.

I do feel that maybe consistent behaviour across all datasets in desired in this case, so I could change the behaviour of Folder to `SAME_AS_TEST` in a separate PR. What do you think? @samet-akcay @ashwinvaidya17

Reply:
hm, if the other benchmarks is correct, then we could leave it for now. I feel that we might need to revisit folder dataset in the future anyway. We currently do not address if the user already has a validation set